extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -191,6 +236,7 @@ export class AmpAd3PImpl extends AMP.BaseElement {
     if (this.layoutPromise_) {
       return this.layoutPromise_;
     }
+    this.lifecycleReporter_.sendPing('adRequestStart');
```

Comment:
No, they differ by the throttling logic.  We don't have a good sense of how much impact that has in practice, so we'd like to capture it.